axum-extra/src/extract/mod.rs

```diff
@@ -56,3 +56,10 @@ pub use crate::json_lines::JsonLines;
 #[cfg(feature = "typed-header")]
 #[doc(no_inline)]
 pub use crate::typed_header::TypedHeader;
+
+#[cfg(feature = "scheme")]
+pub mod scheme;
+
+#[cfg(feature = "scheme")]
+#[doc(no_inline)]
+pub use self::scheme::{Scheme, SchemeMissing};
```

Comment:
Could you please move these declarations up to the groups of module declarations and `self` re-exports before the `json_lines` re-export? Also, the module should not be public.

Reply:
PR is updated; I hope I've done this correctly :)